packages/babel/test/as-input-plugin.js

```diff
@@ -172,7 +171,15 @@ test('generates sourcemap by default', async (t) => {
   } = await bundle.generate({ format: 'cjs', exports: 'auto', sourcemap: true });
 
   const target = 'log';
+
+  // source-map uses the presence of fetch to detect browser environments which
+  // breaks in Node 18
+  const { fetch } = global;
```

Comment:
perhaps `globalThis` would be better here? the CI is running on node 14+ ([here](https://github.com/rollup/plugins/blob/6090005a3da032510e0b45fe4fb603e6d9447461/.github/workflows/validate.yml#L20)) and that standard property is available since node 12

Reply:
Our ESLint does not like `globalThis` yet, so I will stick with `global` for now as I am not yet comfortable messing with the global config.